.github/workflows/ci.yml

```diff
@@ -266,7 +266,7 @@ jobs:
       matrix:
         platform:
           - { str: macos-arm64, arch: arm64, runner: "macos-14" }
-          - { str: macos-x64, arch: x86_64, runner: "macos-latest" }
+          - { str: macos-x64, arch: x86_64, runner: "macos-13" }
```

Comment:
This is high impact enough that we should probably split it out into a separate PR for priority merging.

Reply:
https://github.com/H-uru/Plasma/pull/1586